### PR TITLE
automating Mixer samples

### PR DESCRIPTION
**Pull request by kyessenov**
Moves all Mixer samples to the repo to accommodate automating the documented tasks in tests.
Updates the samples to use the simplified config style.

/assign @geeknoid 
(this is a prerequisite for the documentation change)

### CODE REVIEW

**Comment by kyessenov** [comment]
cc @vadimeisenbergibm @gargnupur for basic sanity checking...
I'm getting a slight headaches after writing this much YAML.

**Comment by kyessenov** [comment]
Contributes to #12773

**Comment by geeknoid** [comment]
Excellent.

/lgtm
/approve

**Comment by kyessenov** [comment]
@duderino Can we merge this? I'll do a final check with the doc change PR, and we can patch samples again if I spot anything wrong.

File: samples/bookinfo/telemetry/metrics-crd.yaml [hunk 1]
```diff
@@ -15,31 +15,29 @@ spec:
...
 apiVersion: "config.istio.io/v1alpha2"
 kind: rule
 metadata:
-  name: doublepromtwo
+  name: doubleprom
   namespace: istio-system
 spec:
   actions:
-  - handler: prometheusdouble
+  - handler: doublehandler.prometheus
```
Comment (gargnupur) [inline samples/bookinfo/telemetry/metrics-crd.yaml]: This can be just "doublehandler" too right?

Comment (kyessenov) [reply samples/bookinfo/telemetry/metrics-crd.yaml]: Actually, no. prometheus qualifier is necessary for non-handlers.

Comment (gargnupur) [reply samples/bookinfo/telemetry/metrics-crd.yaml]: yes, my bad didn't see this is *-crd.yaml file

**Comment by duderino** [comment]
Nice work @kyessenov

**Pull request merged**